databricks_cli/unity_catalog/delta_sharing_cli.py

```diff
@@ -171,6 +187,177 @@ def update_share_cli(api_client, name, new_name, comment, owner,
                       lambda json: UnityCatalogApi(api_client).update_share(name, json))
 
 
+def create_common_shared_data_object_options(f):
+    @click.option('--table', required=True,
+              help='Full name of the table to update from share.')
```

Comment:
These help texts can be updated to apply to both create and update commands, similar to the other UC commands.

Also similar to other UC commands, `--table` help text can be different between create and update.